tools/src/test/scala/scala/scalanative/linker/LinktimeConditionsSpec.scala

```diff
@@ -6,7 +6,6 @@ import org.junit.Assert._
 import scala.scalanative.OptimizerSpec
 import scala.scalanative.build.{Config, NativeConfig}
 import scala.scalanative.nir.{Global, Sig, Type, Val, Rt, Unmangle}
-import scala.scalanative.api.CompilationFailedException
```

Comment:
Moved tests that fails to compile at frontend into `nscplugin`,
Otherwise, we have to parse the `stderr` from Scala compiler spawned in `NIRCompiler.scala`.